Schema modifications for CO2e (#436)
BuildingSync/schema: proposals/2022/Add Emissions.md

# Add Emissions Fields

## Overview

> This proposal is to add several Emissions related fields to the BuildingSync schema to capture information on GHG emissions, specifically CO2e in units of MtCO2e. 

Reviewer: ```suggestion
This proposal is to add several Emissions related fields to the BuildingSync schema to capture information on GHG emissions, in units of MtCO2e. 
```

# Add Emissions Fields

## Overview

> This proposal is to add several Emissions related fields to the BuildingSync schema to capture information on carbon emissions, specifically CO2e in units of kg of CO2e. 

Reviewer: ```suggestion
This proposal is to add several Emissions related fields to the BuildingSync schema to capture information on GHG emissions, specifically CO2e in units of kg of CO2e. 
```

1. Added to AllResourceTotal:

> 	- AnnualAverageCarbonEmissions

Reviewer: ```suggestion
	- AnnualAverageGHGEmissions
```

> 	- AnnualMarginalCarbonEmissions

Reviewer: ```suggestion
	- AnnualMarginalGHGEmissions
```

1. Added to ResourceUse -> Emissions -> Emission:
	- EmissionsLinkedTimeSeriesIDs -> EmissionsLinkedTimeSeriesID
1. Added to TimeSeries -> TimeSeriesReadingQuantity
	- Enum "Emissions"
1. Scenario Type changes by scenario_type:
	1. Added to ScenarioType -> CurrentBuilding, ScenarioType -> Benchmark, and ScenarioType -> Derived Model
		- no changes
	1. Added to ScenarioType -> Target, ScenarioType -> PackageOfMeasures, and ScenarioType -> Other:

> 		- AnnualSavingsAverageCarbonEmissions

Reviewer: ```suggestion
		- AnnualSavingsAverageGHGEmissions
```

> 		- AnnualSavingsMarginalCarbonEmissions

Reviewer: ```suggestion
		- AnnualSavingsMarginalGHGEmissions
```

## Justification

> The BuildingSync schema needs to be modify to accommodate a standard way of reporting GHG emissions (in kg CO2e) to support the focus on decarbonization efforts.

Author: ```suggestion
The BuildingSync schema needs to be modified to accommodate a standard way of reporting GHG emissions (in kg CO2e) to support the focus on decarbonization efforts.
```

> The BuildingSync schema needs to be modify to accomodate a standard way of reporting carbon emissions (in kg CO2e) to support the focus on decarbonization efforts.

Reviewer: ```suggestion
The BuildingSync schema needs to be modify to accommodate a standard way of reporting GHG emissions (in kg CO2e) to support the focus on decarbonization efforts.
```

## Implementation

```xml

>  <xs:element name="AnnualSavingsAverageCarbonEmissions">

Reviewer: ```suggestion
 <xs:element name="AnnualSavingsAverageGHGEmissions">
```

    <xs:annotation>

>       <xs:documentation>Average carbon emissions savings per year. (kg CO2e/year)</xs:documentation>

Reviewer: ```suggestion
      <xs:documentation>Average GHG emissions savings per year. (kg CO2e/year)</xs:documentation>
```

    </xs:annotation>
    <xs:complexType>
      <xs:simpleContent>
        <xs:extension base="xs:decimal">
          <xs:attribute ref="auc:Source"/>
        </xs:extension>
      </xs:simpleContent>
    </xs:complexType>
  </xs:element>

>   <xs:element name="AnnualSavingsMarginalCarbonEmissions">

Reviewer: ```suggestion
  <xs:element name="AnnualSavingsMarginalGHGEmissions">
```

    <xs:annotation>

>       <xs:documentation>Marginal carbon emissions savings per year. (kg CO2e/year)</xs:documentation>

Reviewer: ```suggestion
      <xs:documentation>Marginal GHG emissions savings per year. (kg CO2e/year)</xs:documentation>
```
